datafusion/expr/src/expr_rewriter.rs

```diff
@@ -493,24 +476,18 @@ pub fn replace_col(e: Expr, replace_map: &HashMap<&Column, &Column>) -> Result<E
 /// For example, if there were expressions like `foo.bar` this would
 /// rewrite it to just `bar`.
 pub fn unnormalize_col(expr: Expr) -> Expr {
-    struct RemoveQualifier {}
-
-    impl ExprRewriter for RemoveQualifier {
-        fn mutate(&mut self, expr: Expr) -> Result<Expr> {
-            if let Expr::Column(col) = expr {
-                // let Column { relation: _, name } = col;
-                Ok(Expr::Column(Column {
-                    relation: None,
-                    name: col.name,
-                }))
-            } else {
-                Ok(expr)
-            }
+    rewrite_expr(expr, |expr| {
+        if let Expr::Column(col) = expr {
+            // let Column { relation: _, name } = col;
```

Comment:
Can we remove it?

Reply:
good idea - it looks like it is left over from some previous refactoring:


```suggestion
```